src/Microsoft.VisualStudio.LanguageServices.Razor/ITagHelperResolver.cs

```diff
@@ -4,11 +4,12 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Razor;
 
 namespace Microsoft.VisualStudio.LanguageServices.Razor
 {
-    public interface ITagHelperResolver
+    public interface ITagHelperResolver : ILanguageService
```

Comment:
Can we make a new interface for this in `Microsoft.CodeAnalysis.Razor` namespace (`Microsoft.AspNetCore.Razor.Workspaces` assembly).

I don't want to roslynize this with the wrong API